charts/plex-media-server/Chart.yaml

```
@@ -3,9 +3,9 @@ name: plex-media-server
 description: A Helm chart for deploying a PMS server to a kubernetes cluster
 
 keywords:
-- plex
-- pms
-- Personal Media Server
+  - plex
+  - pms
+  - Personal Media Server
```

nit: no indentation on array keys.

```suggestion
- plex
- pms
- Personal Media Server
```